daprdocs/content/en/operations/observability/tracing/otel-collector/open-telemetry-collector-jaeger.md

```diff
@@ -1,27 +1,30 @@
 ---
 type: docs
-title: "Using OpenTelemetry Collector to collect traces to send to Jaeger"
-linkTitle: "Using the OpenTelemetry for Jaeger"
+title: "Using OpenTelemetry to send traces to Jaeger V2"
+linkTitle: "Using OpenTelemetry for Jaeger V2"
 weight: 1200
-description: "How to push trace events to Jaeger distributed tracing platform, using the OpenTelemetry Collector."
+description: "How to push trace events to Jaeger V2 distributed tracing platform using OpenTelemetry protocol."
 ---
 
-While Dapr supports writing traces using OpenTelemetry (OTLP) and Zipkin protocols, Zipkin support for Jaeger has been deprecated in favor of OTLP. Although Jaeger supports OTLP directly, the recommended approach for production is to use the OpenTelemetry Collector to collect traces from Dapr and send them to Jaeger, allowing your application to quickly offload data and take advantage of features like retries, batching, and encryption. For more information, read the Open Telemetry Collector [documentation](https://opentelemetry.io/docs/collector/#when-to-use-a-collector).
+Dapr supports writing traces using the OpenTelemetry (OTLP) protocol, and Jaeger V2 natively supports OTLP, allowing Dapr to send traces directly to a Jaeger V2 instance. This approach is recommended for production to leverage Jaeger V2's capabilities for distributed tracing.
+
 {{< tabpane text=true >}}
 
 {{% tab "Self-hosted" %}}
-<!-- self-hosted -->
-## Configure Jaeger in self-hosted mode
+## Configure Jaeger V2 in self-hosted mode
 
 ### Local setup
 
 The simplest way to start Jaeger is to run the pre-built, all-in-one Jaeger image published to DockerHub and expose the OTLP port:
 
 ```bash
-docker run -d --name jaeger \
-  -p 4317:4317  \
+docker run --rm --name jaeger \
```

Comment:
I suggest two updates here. 
-  Add that ` -p 9411:9411` clashes with zipkin that may already be running and you should stop zipkin first with a `docker stop` command

- Add the `-d` command so this runs detached in the background. Like this

```
docker run -d --rm --name jaeger \
  -p 16686:16686 \
  -p 4317:4317 \
  -p 4318:4318 \
  -p 5778:5778 \
  -p 9411:9411 \
  cr.jaegertracing.io/jaegertracing/jaeger:2.11.0
```